eZ/Bundle/EzPublishRestBundle/Tests/Functional/TestCase.php

```diff
@@ -226,6 +226,7 @@ private static function clearCreatedElement(array $contentArray)
     protected function createFolder($string, $parentLocationId)
     {
         $string = $this->addTestSuffix($string);
+        $remoteId = md5(uniqid($string, true));
```

Comment:
backport? :)

Reply:
> backport? :)

I guess it wouldn't hurt ;) But for 6.7 or 6.13?

Reply:
6.7 backport #2181